sdk/identity/identity/review/identity.api.md

```diff
@@ -31,6 +31,15 @@ export class AuthenticationError extends Error {
 // @public
 export const AuthenticationErrorName = "AuthenticationError";
 
+// @public
+export interface AuthenticationRecord {
```

Comment:
`AuthenticationRecord` should have a `clientId` property as well as well.

Reply:
It comes from https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-common/src/account/AccountInfo.ts#L14-L20

Should we take in `clientId` and repackage it for MSAL as one of their fields?

Reply:
This data should be returned in the response from `acquireToken***`.

Reply:
Actually, I take that back. In .NET we take that data into the constructor of the `InteractiveBrowserCredential` and store it as a property, `ClientId` on the instance:

https://github.com/Azure/azure-sdk-for-net/blob/a8a0e8cdf5528017fa3dfd22cae81a879e7f0d39/sdk/identity/Azure.Identity/src/InteractiveBrowserCredential.cs#L75-L77

Then when we interactively authenticate we update the authentication record also passing in the stored `ClientId`:

https://github.com/Azure/azure-sdk-for-net/blob/a8a0e8cdf5528017fa3dfd22cae81a879e7f0d39/sdk/identity/Azure.Identity/src/InteractiveBrowserCredential.cs#L207-L214

Reply:
Not quite sure I follow. Are you saying you *do* repack the value for MSAL and rename the field? Or just that .NET works differently than msal-node